cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl

```diff
@@ -1,7 +1,6 @@
 <#ftl ns_prefixes={"xmi":"http://schema.omg.org/spec/XMI/2.1"}>
 <#include '/functions.ftl'>
 <#assign name = elemDoc["self::node()/@name"]>
-<#assign connectors = doc["xmi:XMI/xmi:Extension/connectors/connector"]>
```

Comment:
Have you tested this? I see that you removed the variable `connectors` but then you use it on line 14:
`<#list connectors as connector>`

Reply:
I moved the assignment to functions.ftl so you don't have to assign it every time you want to use them

Reply:
Good idea 👍